content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -14,8 +14,22 @@ problem you are experiencing. See
 the [application troubleshooting guide](/docs/tasks/debug/debug-application/) for tips on application debugging.
 You may also visit the [troubleshooting overview document](/docs/tasks/debug/) for more information.
 
+Before upgrading to Kubernetes 1.24, please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).
+
```

Comment:
The current `main` branch is for 1.25
I'm also ok with this line not being in the top of this page as there is no other content/context around troubleshooting upgrades. 
Maybe a warning like this (sans version) can go in the [Upgrade a Cluster](https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/). A similar sentence is already in the [Upgrading kubeadm clusters](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) page

```suggestion
Before upgrading to Kubernetes 1.24, please go through [Urgent Upgrade Notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes).

```

Reply:
@reylejano Thank you for the suggestions! I'd be committing the changes soon.